apps/backend/src/libs/modules/server-application/base-server-application.ts

```diff
@@ -28,9 +31,15 @@ type Constructor = {
 	config: Config;
 	database: Database;
 	logger: Logger;
+	options: Options;
 	title: string;
 };
 
+type Options = {
+	token: Token;
+	userService: UserService;
+};
+
```

Comment:
Let's pass it this way
```suggestion
	token: Token;
	services: {
	  userService: UserService;
	},
	title: string;
};
```